File: tmp-mei/fgenjs5030319945734616766/base/test/FTestRunner_0.js
import { base$$F_2 } from "../../base/F_2.js";
import * as rt from "../rt/main.js";

export class base$$test$$FTestRunner_0 extends base$$F_2 {
  static $self = new base$$test$$FTestRunner_0();

  $hash$read(sys_m$) {
  return base$$test$$FTestRunner_0.$hash$read$fun(sys_m$, this);
}

  static $hash$read$fun(sys_m$, $this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

}

File: tmp-mei/fgen4255027576711118260/base/test/FTestRunner_0.java
package base.test;
public interface FTestRunner_0 extends base.F_2{
FTestRunner_0 $self = new FTestRunner_0Impl();
base.test.TestRunner_0 $hash$read(Object a_m$);
static base.test.TestRunner_0 $hash$read$fun(base.caps.System_0 sys_m$, base.test.FTestRunner_0 $this) {
  return ((base.test.TestRunner_0)(switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (base.test.TestRunner_0) null;
}})
);
}
}